src/pages/try.js

```diff
@@ -30,7 +30,19 @@ const bs = text => {
   }
 }
 
-const defaultSource = 'let x = 10;\nJs.log x;'
+const decodeBase64 = () => {
+    let url_encode = window.location.href;
+    let url = new URL(url_encode);
+    let digestCode = url.searchParams.get("digest");
```

Comment:
This isn't available on IE unfortunately; also, the getter will return an already encoded snippet, so when you persist "foo bar" into the URL, here you'd get back "foo+bar". Which is then decoded into an invalid snippet.